Assets/LeapMotion/Core/Scripts/LeapServiceProvider.cs

```diff
@@ -292,6 +267,31 @@ public enum FrameOptimizationMode {
       resetTransforms();
     }
 
+    private void checkIsHeadMounted() {
+      bool unused;
+      checkIsHeadMounted(out unused);
+    }
+
+    private void checkIsHeadMounted(out bool shouldEnable) {
+      if (UnityEngine.VR.VRSettings.enabled) {
+        var parentCamera = GetComponentInParent<Camera>();
+        if (parentCamera != null && parentCamera.stereoTargetEye != StereoTargetEyeMask.None) {
+
+          if (!_isHeadMounted) {
+            if (Application.isPlaying) {
+              Debug.LogError("VR is enabled and the LeapServiceProvider is the child of a "
+                           + "camera targeting one or both stereo eyes; You should "
+                           + "check the isHeadMounted option on the LeapServiceProvider "
+                           + "if the Leap is mounted or attached to your VR headset!",
+                             this);
+            }
+            shouldEnable = true;
+          }
+        }
+      }
+      shouldEnable = false;
```

Comment:
There are no other ways this function can exit right?  Which would mean this method can only ever exit with `shouldEnable` equal to false.  The line 288 does nothing.